scripts/kconfig/kconfigfunctions.py

```diff
@@ -875,6 +876,35 @@ def substring(kconf, _, string, start, stop=None):
     else:
         return string[int(start):]
 
+def int_add(kconf, _, arg0, arg1):
+    """
+    Calculate the addition of integers.
+    """
+    return str(int(int(arg0) + int(arg1)))
```

Comment:
It might make more sense to have a `sum` function that takes an arbitrary amount of argument.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/73489#issuecomment-2256289192

I have made it possible to handle multiple arguments using a functional language approach.

Since it is a set of arithmetic operations, if we had to choose one, I think 'add' is better than 'sum'.